src/main/java/com/griddynamics/qa/Calculator.java

```diff
@@ -0,0 +1,11 @@
+package com.griddynamics.qa;
+
+class Calculator {
```

Comment:
As far this is a utility class it is necessary to declare a private constructor.